src/switch_rtp.c

```diff
@@ -9231,7 +9358,7 @@ static int rtp_common_write(switch_rtp_t *rtp_session,
 			my_host = switch_get_addr(bufc, sizeof(bufc), rtp_session->local_addr);
 
 			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG_CLEAN(rtp_session->session), SWITCH_LOG_CONSOLE,
-							  "W %s b=%4ld %s:%u %s:%u %s:%u pt=%d ts=%u seq=%u m=%d\n",
+							  "W WW %s b=%4ld %s:%u %s:%u %s:%u pt=%d ts=%u seq=%u m=%d\n",
```

Comment:
Is this typo?

Reply:
It's intentional, to make it different from otherwise similar message at different line.